cmd/user-agent/src/pages/Interop.vue

```diff
@@ -119,7 +119,24 @@ SPDX-License-Identifier: Apache-2.0
             },
             getVC: async function () {
                 this.clearResults()
-                const credentialQuery = JSON.parse('{"web": {"VerifiableCredential": {}}}');
+                const credentialQuery = {
+                    "web": {
+                        "VerifiableCredential": {
+                            query: [
+                                {
+                                    type: "QueryByExample",
+                                    credentialQuery: {
+                                        reason: "Please present a credential for JaneDoe.",
```

Comment:
JaneDoe?

Reply:
this is test page, now exposed in dashboard

Reply:
used it from sample docs